scripts/docker/buildpack-deps/Dockerfile.ubuntu1804

```diff
@@ -89,17 +89,16 @@ RUN set -ex; \
 	rm -f $TGZFILE;
 
 # HERA
-ARG HERA_HASH="622663cb3bc1fa07213c6e1fe8070c5c259096e75039a46d014b2a3448b5cf44"
-ARG HERA_MAJOR="0"
-ARG HERA_MINOR="3"
-ARG HERA_MICRO="0"
 RUN set -ex; \
-	HERA_VERSION="$HERA_MAJOR.$HERA_MINOR.$HERA_MICRO"; \
-	TGZFILE="hera-$HERA_VERSION-linux-x86_64.tar.gz"; \
-	wget https://github.com/ewasm/hera/releases/download/v$HERA_VERSION/$TGZFILE; \
-	sha256sum $TGZFILE; \
-	tar xzpf $TGZFILE -C /usr; \
-	rm -f $TGZFILE;
+	cd /usr/src; \
+	git clone --branch="v0.3.1" --recurse-submodules https://github.com/ewasm/hera.git; \
```

Comment:
I'd suggest using `--depth 1` in clone commands as a general rule unless there's a need to have all history available.